meta-genivi-dev/meta-sdl/conf/layer.conf

```diff
@@ -0,0 +1,13 @@
+# We have a conf and classes directory, add to BBPATH
+BBPATH .= ":${LAYERDIR}"
+
+# We have recipes-* directories, add to BBFILES
+BBFILES += "${LAYERDIR}/recipes-*/*/*.bb \
+	${LAYERDIR}/recipes-*/*/*.bbappend"
+
+BBFILE_COLLECTIONS += "genivi-dev-meta-sdl-append"
+BBFILE_PATTERN_genivi-dev-meta-sdl-append = "^${LAYERDIR}/"
+BBFILE_PRIORITY_genivi-dev-meta-sdl-append = "10"
+
+LAYERSERIES_COMPAT_genivi-dev-meta-sdl-append = "sumo"
+
```

Comment:
Oh... No it was basically unintentional. I was doing the work on the sumo branch and this was an effect of me rebasing it onto master instead.  If we merge sumo first and then rebase this then it should cause a conflict most likely, and it can be then fixed up.  I'll try to monitor that.